src/main/java/com/buttercms/example/controllers/PostController.java

```diff
@@ -0,0 +1,25 @@
+package com.buttercms.example.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.buttercms.IButterCMSClient;
+import com.buttercms.model.PostResponse;
+
+@Controller
+public class PostController {
+  private IButterCMSClient butterCMSClient;
+
+  public PostController(IButterCMSClient butterCMSClient) {
+    this.butterCMSClient = butterCMSClient;
+  }
+
+  @RequestMapping("/post/{slug}")
```

Comment:
Can we make blog home exist at `/blog/` and blog post at `/blog/{slug}`?

Blog home template would contain this chunk of code to display their list of blog posts:

```
<h3>Here is a list of Posts</h3>
<ul>
    <c:forEach items="${posts}" var="post">
        <li><a href="/post/${post["slug"]}">${post["title"]}</a></li>
    </c:forEach>
</ul>
```